docs/scripts/filter.js

```diff
@@ -1,26 +1,38 @@
 function applyFilters(){
-    brandFilter();
+    var allSections = ["brand", "article", "gender"]
+    allFilter(allSections);
+    console.log(filters);
 }
 
-function brandFilter(){
-    var allBoxes = document.getElementsByClassName("brand");
-    var allChecked = document.getElementsByClassName("brandAll");
-    var checked = [];
-    if (allChecked[0].checked){
-        for (var j=0; j < allBoxes.length; j++){
-            allBoxes[j].checked = true;
+function allFilter(sections){
```

Comment:
This algo is off. It works properly but if you click twice it will double up on filters in the array. You need to check if the filter is already in the array before adding it or maybe just reset the json and start over after clicking filter. Lemme know if that makes sense.